socorro/processor/rules/general.py

```diff
@@ -37,14 +38,43 @@ def de_null(self, s):
         return s
 
     def action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
+        had_nulls = False
+
         # Go through the raw crash and de-null keys and values
-        for key, val in raw_crash.items():
+        for key, val in list(raw_crash.items()):
```

Comment:
It's possible for this rule to change the thing being iterated over. Given that, we need to convert the iterator to a new list that doesn't change.